website/content/en/status/report-2023-04-2023-06/puppet.adoc

```diff
@@ -0,0 +1,16 @@
+=== Puppet
+
+Links: +
+link:https://puppet.com/docs/puppet/latest/puppet_index.html[Puppet] URL: link:https://puppet.com/docs/puppet/latest/puppet_index.html[] +
+
+Contact: Puppet Team <puppet@FreeBSD.org>
+
+Puppet is a Free Software configuration management tool, composed of a source of trust (Puppet Server) that describes the expected configuration of machines with a domain-specific language, and an agent (Puppet Agent) on each node which enforce the actual configuration match the expected one. An optional database (PuppetDB) can be setup for reporting and describing advanced schemas where the configuration of a machine depends on the configuration of another one.
```

Comment:
s/which enforce the actual configuration match the expected one/which enforces that the actual configuration matches the expected one/